docs/source/tutorials-basic/driving.rst

```diff
@@ -0,0 +1,197 @@
+gets executed. Notice that :py:meth:`~easygopigo3.EasyGoPiGo3.drive_cm` is a blocking function in this case.
+
+.. code-block:: python
+
+    print("Drive the motors 50 cm and then stop.")
+    gpg.drive_cm(50, True)
+    time.sleep(1)
+
+
+Then right for a second.
+
+.. code-block:: python
+
+    print("Turn right 1 second.")
+    gpg.right()
+    time.sleep(1)
+
+And likewise to the left.
+
+.. code-block:: python
+
+    print("Turn left 1 second.")
+    gpg.left()
+    time.sleep(1)
+
+Finally, stop the robot.
+
+.. code-block:: python
+
+    print("Stop!")
+    gpg.stop()
+    print("Done!")
+
+If you want to run this by yourself, `here's the script on github <https://github.com/DexterInd/GoPiGo3/blob/master/Software/Python/Examples/easy_Motors.py>`_.
+
+.. image:: https://i.imgur.com/xWzt6c4.gif
+
+=======================
+Describing a Square
+=======================
+
+To make the GoPiGo3 describe a square by moving itself, you need to run the following script.
+To do this, :py:meth:`~easygopigo3.EasyGoPiGo3.drive_cm` and :py:meth:`~easygopigo3.EasyGoPiGo3.turn_degrees`
+methods are required. A square with the side length of 30cm is drawn. The square is drawn clockwise.
+
+.. code-block:: python
+
+   from easygopigo3 import EasyGoPiGo3
+
+   gpg = EasyGoPiGo3()
+   length = 30
+
+   for i in range(4):
+     gpg.drive_cm(length) # drive forward for length cm
+     gpg.turn_degrees(90) # rotate 90 degrees to the right
+
+.. image:: https://i.imgur.com/6Q7uHq9.gif
+
+=====================
+Making Circular Moves
+=====================
+
+Driving straight in one direction is one thing, but rotating around a center axis at a specific radius is
+something entirely different. In this example, the GoPiGo3 draws half a circle and then returns
+on the same track by spinning itself on the spot.
+
+The radius of the circle is set at 50 centimeters and the robot will move for half of the circle (aka 180 degrees).
+
+.. code-block:: python
+
+   from easygopigo3 import EasyGoPiGo3
+
+   gpg = EasyGoPiGo3()
+
+   gpg.orbit(180, 50) # draw half a circle
+   gpg.turn_degrees(180) # rotate the GoPiGo3 around
+   gpg.orbit(-180, 50) # return on the initial path
+   gpg.turn_degrees(180) # and put it in the initial position
+
+.. image:: https://i.imgur.com/cfdbsID.gif
+
+====================
+Drawing an *8* Shape
+====================
+
+Let's say we want to draw an 8 shape with the GoPiGo3 and at the end have the GoPiGo3 reach the same position
+it initially left from.
+
+To do this, we have to use :py:meth:`~easygopigo3.EasyGoPiGo3.orbit` and :py:meth:`~easygopigo3.EasyGoPiGo3.drive_cm` methods.
+
+.. code-block:: python
+
+   from easygopigo3 import EasyGoPiGo3
+
+   gpg = EasyGoPiGo3()
+   radius = 30
+
+   gpg.orbit(-270, radius) # to rotate to the left
+   gpg.drive_cm(radius * 2) # move forward
+   gpg.orbit(270, radius) # to rotate to the right
+   gpg.drive_cm(radius * 2) # move forward
+
+.. image:: https://i.imgur.com/SQrRBd8.gif
```

Comment:
too bad we don't have a pen attachment. It would be neat to show the 8 shape!

Reply:
I've sped up the GIF so that you get the illusion of an 8 shape.